scripts/corfu_server.sh

```diff
@@ -37,7 +37,7 @@ fi
 
 
 # default heap for corfudb
-CORFUDB_HEAP="${CORFUDB_HEAP:-1000}"
+CORFUDB_HEAP="${CORFUDB_HEAP:-2000}"
```

Comment:
Can you just read the vm option from the logunit server and compute a ratio for the cache. That is a small enough change.